dags/hivemind_etl_helpers/src/db/discord/utils/fetch_channel_thread_name.py

```
@@ -0,0 +1,25 @@
+        if channel is None:
+            return None
+        else:
+            return channel["name"]
+
+    def fetch_discord_thread_name(self, thread_id: str) -> str:
+        query = {"id": thread_id}
+        thread = self.threads_collection.find_one(query)
+        if thread is None:
+            return None
+        else:
+            return thread["name"]
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)